staff/marti-malek/cv/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Martí Malek</h1>
+        <img src="superrandom" alt="My Image">
+        <h3>Personal Information</h3>
+        <address>
+        Address: C/Pedrell N.80 <b>Barcelona</b><br>
+        Tlf: 697 501 950<br>
+        e-mail: <a href="mailto:martimaledot@hotmail.es">martimalekdot@hotmail.es</a>
+        </address>
+        <h3>Profile</h3>
+        <p><i>Very hard-working person with energy and skills</i></p>
+    </header>
+    <section>
+        <h2>Experience</h2>
+        <ul>
+            <li>Volunteer in <a href="https://www.bancdelsaliments.org/es/"><b>Banc dels Aliments</b></a> in 2018</li>
+            <li>18 years of expertise as a day-dreamer</li>
+            <li>More than 10 years of expertise as a person</li>
+        </ul>
+    </section>
+    <section>
+        <h2>Education</h2>
+        <ul>
+            <li>High school</li>
+            <li>Currently doing a Full-Stack Web Development Bootcamp in <a href="https://www.skylabcoders.com/es"><b>SkylabCodersAcademy</b></a></li>
+            <li>Currently studying chinese in <a href="https://www.eoibd.cat/es/"><b>Escola Oficial d'Idiomes</b></a></li>
```

Comment:
This is to style it! [https://www.w3schools.com/tags/tag_b.asp](url)

Reply:
Fixed it! ;)